### PR TITLE
Updated the New York Times pages clone project with provided requirement.html

### PR DESCRIPTION
**Pull request by mahbubul14**
# Feature I used here:

- fixed header
- navbar
- body
- social media
- button
- footer

# The following files are also included:

- index.html file - this contains the html elements
- style.css file - for styling the html file
- README file - for highlighting some description about the project

### CODE REVIEW

**Review by teekaytech — CHANGES_REQUESTED** [review]
### Status: Invalid Code Review Request :x: 

__Hi @mahbubul14, I commend you for the great effort put into this project. However, I have to mark the project as invalid due to the following reasons:__

- [x] Your GitHub Actions checks are not running: to consider this project ready for review, you need to ensure that all GitHub Action Checks are running/passing successfully. 
- [x] You should ensure that your PR has a descriptive title and summary. 
- [x] You should also present a descriptive Readme for your project. [This template ](https://github.com/microverseinc/readme-template)might help.


Friendly reminder: You should make sure that your repo looks professional before review.
Please take a step back and read a lesson [here](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/).
Do not forget to apply 3 simple rules from this lesson (it will be easy as you can use a template provided there). This Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready!


* * *
**Please, do not open a new Pull Request for subsequent reviews. You should use the same Pull Request submitted for the first review unless it is requested otherwise.**

**Always post your inquiries and questions on this PR thread as a comment. You can mention me (@teekaytech) so I can see your comments and respond.**

* * * 

**_You can reach out to me on Twitter (@ola_lere), Github(@teekaytech), or slack (@taofeekolalere )_**

Happy Coding :man_technologist:

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello @mahbubul14 :wave:

__You've really done well so far. Here're the major highlights of this PR:__

- PR title and summary is descriptive :ok_hand: 
- Readme has been improved :+1: 
- All linter checks running/passing successfully :100: 

__However, some changes are required before this project can be approved. Please check my comments below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

- [x] There seems to be a bug with your deployment: your readme is being rendered instead of your webpage. This may be caused by your naming convention: you should name your webpage (the HTML file `Homepage.html`) to index.html. This will allow the GitHub page configuration to identify your homepage.
- [x] The `MIT license` link on your homepage seems to be broken. Kindly fix it.
- [x] Kindly work on the alignment of the items on your navbar: the Hamburger, Search, and the linked text should be aligned to the leftmost side, the NYT logo should be at the center, while the `Subscribe Now` and `Log In` buttons should be aligned to the rightmost side. 
**Original navbar**
original
**Cloned navbar**
clone

- [x]  The banner image should be wider than the remaining contents of the page.
**Original page**
original
**Cloned page**
clone2
- [x] Your `Theory of Inflation` section is incomplete: you need to put the texts directly on top of the image, as we have on the original page. You should also observe the width of this section, as it's also wider than the other contents. Hint: You can use the `CSS grid` in this section.
**Original Page**
original
**Cloned Page**
clone
- [x] Some of the `Suggested newsletters for you`, `Most Popular`, `More in Space and Astronomy` and `Editors' Picks` sections are not implemented while others are not well-aligned. Kindly ensure that these sections of your page look similar to what we have on the original page.
**Original Page**
original
- [x] Improvement is also required in your bottom navigation section.
- Ensure that the structure looks similar. Observe the difference in the images below and make necessary changes.
**Original**
original
**Clone**
clone
- All items in the bottom navigation section should be linked (i.e. use `<a>` tag around each of the items). Any valid value e.g. "#" is okay for each of their `href` attributes.
- [x] The `Editor's Picks` section on the rightmost side is missing. Kindly implement it.
**Editor's Picks**
original



**[ OPTIONAL CHANGES ]**

- [x] Use descriptive and meaningful class names as your CSS selectors. Check [this article](https://speckyboy.com/good-bad-css-practices) on common CSS bad practices and always avoid them.

* * *
**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review unless it is requested otherwise.**

**Always post your inquiries and questions on this PR thread as a comment. You can mention me (@teekaytech) so I can see your comments and respond.**

**REMINDER: As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 code reviews left for this project.**

_**Please make the required changes and submit another review request when you are done.**_

* * *

**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

File: index.html [hunk 1]
```diff
@@ -0,0 +1,521 @@
...
+        <h1>Space Ripples Reveal Big Bang’s Smoking Gun</h1>
+        <div class="social-media">
+            <img src="/image/facebook-f-brands.svg" alt="image">
+            <img src="image/whatsapp-brands.svg" alt="image">
+            <img src="/image/twitter-brands.svg" alt="image">
+            <img src="image/envelope-solid.svg" alt="image">
+            <img src="image/share-solid.svg" alt="image"> 
+            <img src="image/bookmark-regular.svg" alt="image">
+            <img src="image/comment-alt-regular.svg" alt="image">
+        </div>
```
Comment (Berabjesus) [inline index.html]: ### main page 👇 
m1
### your page 👇 
err1

**As you can see the deployed and the local version of your page is messed up on the social medial links section**

- [x] Please take a look at the code below and fix the image path for all the icons
```
// Bad Paths
<link rel="stylesheet" href="/assets/stylesheets/main.css">
<link rel="stylesheet" href="/assets/js/bootstrap.min.js">
<img src="/imgs/myimage.png">


// Good Paths
<link rel="stylesheet" href="./assets/stylesheets/main.css">
<link rel="stylesheet" href="./assets/js/bootstrap.min.js">
<img src="./imgs/myimage.png">
```

- [x] Make sure the images are links
- [x] Give the icons proper styling. They are currently overflowing to the right.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,521 @@
...
+        <div>
+            <img src="/image/1st.jpg" alt="image" class="h1img">
```
Comment (Berabjesus) [inline index.html]: - [x] Please take a look at the suggestion above and fix the image path for all the image.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,521 @@
...
+        <p class="content">
+            The ripples manifested themselves as faint spiral patterns in a bath of microwave radiation that permeates space and preserves a picture of the universe when it was 380,000 years old and as hot as the surface of the sun.
+        </p>
+        <p class="content">
+            Dr. Kovac and his collaborators, working in an experiment known as Bicep, for Background Imaging of Cosmic Extragalactic Polarization, reported their results in a scientific briefing at the Center for Astrophysics here on Monday and in a set of papers submitted to The Astrophysical Journal.
+        </p>
+        <br><br>
+        <!-- inflation section here -->
+        <div class="inflation-container">
+            <h4>The Theory of Inflation</h4>
```
Comment (Berabjesus) [inline index.html]: - [x] Make sure 'The Theory of Inflation' section is centered horizontally. It is currently leaning to the left edge of the browser.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,521 @@
...
+            <div class="social-media-bottom">
+                <img src="/image/facebook-f-brands.svg" alt="image">
+                <img src="image/whatsapp-brands.svg" alt="image">
+                <img src="/image/twitter-brands.svg" alt="image">
+                <img src="image/envelope-solid.svg" alt="image">
+                <img src="image/share-solid.svg" alt="image">
+            </div>
+        </div>
+        <br><br>
+        <!-- suggested newletter section -->
```
Comment (Berabjesus) [inline index.html:331]: ### Main Page 👇 
image

### Your page 👇 
image

- [x] Please use the suggestions above to fix the image paths for this section and all its image contents too
- [x] Make sure this section is centered horizontally.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,521 @@
...
+        <div class="bottom-social">
+        <button class="comments-btn" type= "button"> READ 615 COMMENTS</button>
+            <div class="social-media-bottom">
+                <img src="/image/facebook-f-brands.svg" alt="image">
+                <img src="image/whatsapp-brands.svg" alt="image">
+                <img src="/image/twitter-brands.svg" alt="image">
+                <img src="image/envelope-solid.svg" alt="image">
+                <img src="image/share-solid.svg" alt="image">
+            </div>
+        </div>
```
Comment (Berabjesus) [inline index.html]: ### Main page 👇 
image

### your page 👇 
image

- [x] Please work on the styling and the proper image path as specified above.

**Review by Berabjesus — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 
Hi @mahbubul14  An awesome start :+1: ! Very good README :100: Good job with the professional PR 👍 
### However, You have a few changes to carry out before we proceed so please take a look at the things you have to work on and their respective review comments below.

- [x] Please make sure the NYT logo is a link.
- [x] The social media links are not correctly linked and styled.
- [x] Please use proper semantic HTML to avoid [Div soup](https://www.hackterms.com/div%20soup)
- [x] The project should be made with a combination of all three techniques ([float](https://css-tricks.com/all-about-floats/), [flex](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) and [grid](https://css-tricks.com/snippets/css/complete-guide-grid/)). ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/positioning_and_floating_elements/flex_float_grid_requirement_pathwright.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59538403/)). You have not used grid
- [x] Links in the footer are not properly tagged as anchor tags
- [x]  CSS rules are repeated. For example `display: flex;` is used eleven times. Instead, create a common class for it and use it in the HTML
- [x] No horizontal scrollbar. 

**Please have a look at the specific comments in the code below**

#### **Please, do not open a new Pull Request for subsequent reviews. You should use the same Pull Request submitted for the first review unless it is requested otherwise.**

#### ** ❗ You have two (2) remaining reviews. As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), when you reach the limit of 3 code reviews and your project was not approved, you will need to repeat the project. If you think that the code review was not fair, you can [request a second-opinion using this form.](https://airtable.com/shrmBIv6cXYrq4Fgv)**

#### **If you have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help** 
**HAPPY CODING :computer:[Bereket Beshane](https://bereketababu.me)** | [LinkedIn](https://linkedin.com/in/bereketbeshane) | [GitHub](https://github.com/Berabjesus)

File: index.html (hunk 1 above)
Comment (mahbubul14) [reply index.html]: @berabjesus,
The icon absolutely looks good at my browser, Why it is so different from your side?

Comment (Berabjesus) [reply index.html]: Hi @mahbubul14, as you can see in the image below from a different browser, although the icons are sized properly, they don't load. The reason is properly explained above so please use the suggestion for the appropriate path to the SVGs.
image

File: index.html [hunk 6]
```diff
@@ -0,0 +1,541 @@
...
+    <link rel="icon" href="./image/newyorktimesicon.png" type="image/gif" sizes="16x16">
+    <script src="https://kit.fontawesome.com/b435954bf0.js" crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.gstatic.com"/>
+</head>
+<body>
+    <header>
+        <nav class="header-section">
+            <i class="fas fa-bars"></i>
+            <i class="fas fa-search"></i>
+            <span class="nav-text">SPACE & COSMOS</span>
```
Comment (Berabjesus) [inline index.html:18]: ## Optional
- [ ] [OPTIONAL] When I first reviewed your project the highlighted sections were links and now the anchor tag is removed. Please make them a link again wrapping them in an anchor tag or using your previous implementation.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,541 @@
...
+            <img src="./image/1st.jpg" alt="image" class="h1img">
+            <p class="imgtext">
+                Alan Guth was one of the first physicists to hypothesize the existence of inflation, which explains how the universe expanded so uniformly and so quickly in the instant after the Big Bang 13.8 billion years ago. <span>Rick Friedman for The New York Times</span>
+            </p>
+        </div>
+        <h4 class= "writter">By <a class="linkblack" href= "#">Dennis Overbye</a></h4>
+        <span class="date">March 17, 2014</span>
+    </div>
+    <article>
+        <p id="p1" class="content"><b>SIGN UP FOR SCIENCE TIMES:</b> Get stories that capture the wonders of nature, the cosmos and the human body.<button class="sign-up" type="button">Sign Up</button> </p>
```
Comment (Berabjesus) [inline index.html]: ## Optional: - When I first reviewed your project the sign up button was correct now it's not styled correctly please correct it.
original 👇 
image

Yours 👇 
image

File: index.html [hunk 8]
```diff
@@ -0,0 +1,541 @@
...
+                <a href="#"><i class="fab fa-facebook-f"></i></a>
+                <a href="#"><i class="fab fa-whatsapp"></i></a>
+                <a href="#"><i class="fab fa-twitter"></i></a>
+                <a href="#"><i class="fas fa-envelope"></i></a>
+                <a href="#"><i class="fas fa-share"></i></a>
+            </div>
+        </div>
+        <br><br>
+        <!-- suggested newletter section -->
+        <h4 class="suggested-header">Suggested newsetter for you</h4>
```
Comment (Berabjesus) [inline index.html:332]: ## Mandatory

- [x] As stated in the previous review, this section should be centered with the appropriate space. Currently, your page's layout is very much close to the edge of the browser.

#### main page 👇 
image

#### yours 👇 
image

## optional 

- [ ] Add the plus icons to the top right corner of the images here

File: index.html [hunk 9]
```diff
@@ -0,0 +1,541 @@
...
+                <a href="#"><img class= "editors-picks-img" src="./image/editor3.jpg" alt="image">
+                <p>Family Travel Gets Complicated Without a Cove Vaccin for Kids</p>
+                <p>March 29</p>
+                </a>
+          </div>
+        </section>
+        </main>
+        <!-- footer section -->
+    <footer>
+        <img src="./image/logo.jpg" alt="image" class="footer-logo">
```
Comment (Berabjesus) [inline index.html:429]: ## Optional 

- [ ] The footer logo should also be a link.

**Review by Berabjesus — CHANGES_REQUESTED** [review]
## Status: REQUIRED CHANGES :warning: :construction: 
#### Hi again @mahbubul14 ! You have done brilliantly to work on the requested changes 👍 👍 
#### However, you have missed just one mandatory change from the previous review. Please work on it and make a request for another code review. Have a look at the  OPTIONAL changes from the comments below too.

#### Mandatory
- [x] Make sure the `suggested newsletter section` is centered and not stretched. Please have a look at the comments on the code below.

#### Optional
- [ ] When I first reviewed your project some of the stylings that were correct (Example: navbar Hamburger button was a link) are now incorrect. Please have a look at the comments on the code below and work on them.

### **Please, do not open a new Pull Request for subsequent reviews. You should use the same Pull Request submitted for the first review unless it is requested otherwise.**

### ** ❗ You have one (1) remaining review. As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), when you reach the limit of 3 code reviews and your project was not approved, you will need to repeat the project. If you think that the code review was not fair, you can [request a second-opinion using this form.](https://airtable.com/shrmBIv6cXYrq4Fgv)**

#### **If you have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help** 
**HAPPY CODING :computer:[Bereket Beshane](https://bereketababu.me)** | [LinkedIn](https://linkedin.com/in/bereketbeshane) | [GitHub](https://github.com/Berabjesus)

**Review by teekaytech — APPROVED** [review]
# Status: Approved :green_circle: 

### Congratulations @mahbubul14  :tada: :clinking_glasses: 

_**Thank you for fixing the mandatory changes highlighted by the last reviewer :handshake:**_

**This project has passed all the mandatory requirements :clap: 
It is therefore approved :heavy_check_mark:**

**Comment(s)**
- In your best interest, you're advised to always strive to implement most (if not all) of the optional changes. These changes are there to make your project look professional.
- You might want to include instructions on how to set up your project in a development environment, on your Readme file. 

Again, congratulations, you can merge to the default branch.
Do not forget to fill the project completion form, as this is important for your weekly progress score.

**_Good luck with your next task._**

* * * 
**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

**Pull request merged**